Detectors/MUON/MCH/Workflow/src/digits-reader-workflow.cxx

```diff
@@ -15,33 +15,12 @@
 /// \author Andrea Ferrero, CEA
```

Comment:
I believe this file should not be modified by this PR, right ?

Reply:
@aphecetche : no it shouldn't
I will now update a version with the file from dev.

Reply:
@mwinn2 best would be to simply exclude it from this PR altogether as it's not supposed to be changed, right ?

Reply:
@aphecetche Ok.